### PR TITLE
test(app): App app settings

### PR DESCRIPTION
**Pull request by reachojhaneha**
This PR covers the automation e2e cases for app-settings 5.1 
App-setting has 4 tabs - general, privacy, advanced and feature flag.

Tickes closes : [general tab](https://app.zenhub.com/workspaces/app--ui-5f6248dba845b000139aa767/issues/opentrons/opentrons/9806) 
[privacy tab](https://app.zenhub.com/workspaces/app--ui-5f6248dba845b000139aa767/issues/opentrons/opentrons/9816)
[advanced tab](https://app.zenhub.com/workspaces/app--ui-5f6248dba845b000139aa767/issues/opentrons/opentrons/9817)
[Feature flag](https://app.zenhub.com/workspaces/app--ui-5f6248dba845b000139aa767/issues/opentrons/opentrons/9818)

### CODE REVIEW

**Review by smb2268 — APPROVED** [review]
Neha and I went over the script in a 1:1 today and it covers all necessary cases.

File: app-testing/gen/electron/buildflags/buildflags.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#define BUILDFLAG_INTERNAL_ENABLE_PDF_VIEWER() (1)
+#define BUILDFLAG_INTERNAL_ENABLE_TTS() (1)
+#define BUILDFLAG_INTERNAL_ENABLE_COLOR_CHOOSER() (1)
+#define BUILDFLAG_INTERNAL_ENABLE_ELECTRON_EXTENSIONS() (1)
+#define BUILDFLAG_INTERNAL_ENABLE_BUILTIN_SPELLCHECKER() (1)
+#define BUILDFLAG_INTERNAL_ENABLE_PICTURE_IN_PICTURE() (1)
+#define BUILDFLAG_INTERNAL_ENABLE_WIN_DARK_MODE_WINDOW_UI() (0)
+#define BUILDFLAG_INTERNAL_OVERRIDE_LOCATION_PROVIDER() (0)
+
+#endif  // ELECTRON_BUILDFLAGS_BUILDFLAGS_H_
```
Comment (y3rsh) [inline app-testing/gen/electron/buildflags/buildflags.h]: What is this file for?

**Review by y3rsh — APPROVED** [review]
Super Nice!  Very impressed how quickly you picked up on the new tools and `Base` object use 😊 
I think this versioning is good for now.

File: app-testing/gen/electron/buildflags/buildflags.h (hunk 1 above)
Comment (reachojhaneha) [reply app-testing/gen/electron/buildflags/buildflags.h]: this is removed as it was added by mistake

**Pull request merged**